the issue with duplicated channels persists (#590)
uyuni-project/uyuni-docs: modules/upgrade/pages/proxy-uyuni.adoc

uyuni-proxy-stable-leap-152
----
. Fully synchronize all channels with [command]``spacewalk-repo-sync``.
  In case of already defined repository URLs, continue with <<uyuni-202007-channeldupes>>.

> [[uyuni-202007-channeldupes]]

Reviewer: There's still this 202007 thing here, right?

Author: yes, I kept it that way. It is just a label or ID.  I usually change those things only if we do a general cleanup (of if I think it is too confusing).

> == Troubleshooting with Duplicated Repository URLs

Reviewer 2: ```suggestion
== Troubleshooting Duplicate Repository URLs
```

// https://github.com/SUSE/spacewalk/issues/12008

> If you want to add two or more repositories with the same URL (client tools for openSUSE Leap 15.0, 15.1, and 15.2) in {productname} with [command]``spacewalk-common-channels``, you will see an error such as:

Reviewer: I see a small problem about this. the Leap channels on SUSE Manager should NOT be used from `spacewalk-common-channels` under any circumstances.

And right now the only "support" OS that uses `spacewalk-common-channels` on SUSE Manager is Debian, and AFAIK it's not affected by this.

Should we maybe have this for Uyuni only?

Author: ok, thanks for the background info.  I think thus far we publish this in the Uyuni context only (but we should be prepared).  I think I must read again what we say about using Leap with suma.

Author: in the client config, Lana put in nice ifdefs.  I think everything is fine and I can merge this one.

> If you want to add two or more repositories with the same URL (client tools for openSUSE Leap 15.0, 15.1, and 15.2) in {productname} with [command]``spacewalk-common-channels``, you will see an error such as:

Reviewer 2: ```suggestion
If you add two or more repositories with the same URL (client tools for openSUSE Leap 15.0, 15.1, and 15.2) in {productname} with [command]``spacewalk-common-channels``, an error like this is shown:
```

. In the upper right corner, click btn:[Manage Channel].

> . Click the [guimenu]``Repositories`` tab, and select [systemitem]``External - Uyuni Client Tools for openSUSE Leap 15.1 (x86_64)``.

Reviewer 2: ```suggestion
. Navigate to the [guimenu]``Repositories`` tab, and select [systemitem]``External - Uyuni Client Tools for openSUSE Leap 15.1 (x86_64)``.
```

. Click btn:[Update Repositories].

. Navigate to menu:Repositories[Sync] subtab, and click btn:[Sync Now].

. Do the same with [systemitem]``Uyuni Proxy Stable for openSUSE Leap 15.2 (x86_64)`` and [systemitem]``External - Uyuni Proxy Stable for openSUSE Leap 15.1 (x86_64)``.

> If you unfold the [systemitem]``openSUSE Leap 15.2 (x86_64)`` channel, you will see all child channels populated with packages.

Reviewer 2: ```suggestion
Expand the [systemitem]``openSUSE Leap 15.2 (x86_64)`` channel to see the child channels populated with packages.
```

. Click btn:[Next], and [guimenu]``Confirm Software Channel Change`` with btn:[Confirm].
. Click menu:Software[Packages > Upgrade], and select all the packages to be updated on the proxy, and then apply the selection.
. Re-register with the reactivation key using the [command]``rhnreg_ks`` command-line utility.

>   The system will be re-registered with the same id, history, and groups.

Reviewer 2: ```suggestion
  The system is re-registered with the same ID, history, and groups.
```
